tests/test_charging_temperature.py

```diff
@@ -0,0 +1,34 @@
+from unittest import TestCase
+
+from src.check_limits import BatteryManagementSystem
+
+
+class TestChargingTemperature(TestCase):
+
+    def setUp(self) -> None:
```

Comment:
The methods should be defined in such a way that it is not aware of  language  environment the software is run in.
The code that prints the messages should pick messages either from English strings or German strings based on language configuration.
In this way, whenever new language support is added, code changes required for addition is minimum